TTT/Game.cpp

```diff
@@ -0,0 +1,90 @@
+#include "Game.hpp"
+
+Game::Game() {
+	ticTacToe();
+}
+
+Game::~Game() {};
+
+void Game::setGameMode() {
+	std::cout << "Please, input integers for game field sides (horizontal and vertical), using space: ";
+	std::cin >> m_hor >> m_vert;
+	m_towin = std::min(m_hor, m_vert);
+}
+
+void Game::setSides(GameField::Value valP1) {
+	valP1 == GameField::Value::CROSS ? m_player2 = GameField::Value::ZERO : m_player2 = GameField::Value::CROSS;
+	m_player1 = valP1;
+}
+
+void Game::swapSides() {
+	(m_player1 == GameField::Value::CROSS) ? setSides(GameField::Value::ZERO) : setSides(GameField::Value::CROSS);
```

Comment:
```suggestion
	setSides(m_player1 == GameField::Value::CROSS) ? GameField::Value::ZERO : GameField::Value::CROSS);
```
Better make a function getOppositeValue() to hide this ternary operator and use it where appropriate.